src/converter/positioning.py

```diff
@@ -100,7 +100,7 @@ def convert(fig_item: dict) -> _Positioning:
 
     return {
         "frame": Rect(
-            constrainProportions=fig_item.get("proportionsConstrained", False),
+            constrainProportions=True if "targetAspectRatio" in fig_item else False,
```

Comment:
Un-related to auto layout/stacks, however, it looks like Figma always sets `proportionsConstrained` to `true`, and it is actually this `targetAspectRatio` coords map that is set when the aspect ratio is locked for a frame/layer/etc